app/views/shopping_shared/tabs/_custom.html.haml

```diff
@@ -0,0 +1,3 @@
+.content
+  .row
+    = @distributor.custom_tab.content.html_safe if @distributor.custom_tab&.content.present?
```

Comment:
We need a way to sanitise the content. Right now, someone could inject <script> tags.

Attack scenario: I open a shop and lure an OFN admin to my custom tab. The script code sends OFN API requests to place paid orders for me, add tags to my customer account or trigger a refund of $1,000,000 for my previous order. Maybe it's not that easy but it would be good to have some peace of mind here.

Oh, Rails has us covered:
https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html

```suggestion
    = @distributor.custom_tab.content.sanitize if @distributor.custom_tab&.content.present?
```

We probably want to list the allowed tags from Trix here.

Reply:
Also I thought the `if` wouldn't be necessary. This partial should only be rendered if `custom_tab` is present, and we can ignore a nil value.
```suggestion
    = @distributor.custom_tab.content&.sanitize
```

Reply:
Also worth pointing out that Trix will probably scrub any script tags before submitting, but we still need to protect against any other method of injecting it into the `content` attribute.

Reply:
Created a sanitizer : 383d9dc161!